lib/Doctrine/DBAL/Migrations/QueryWriter.php

```diff
@@ -4,17 +4,14 @@
 
 namespace Doctrine\DBAL\Migrations;
 
-/**
- * @since  1.6.0
- * @author Luís Cobucci <lcobucci@gmail.com>
- */
 interface QueryWriter
 {
     /**
-     * @param string $path
-     * @param string $direction
-     * @param array $queriesByVersion
-     * @return bool
+     * @param string[][]|string[] $queriesByVersion
```

Comment:
We need to get rid of this later, array of strings or array of arrays of strings doesn't make sense. :see_no_evil: